handlers/delete_session.go

```diff
@@ -0,0 +1,31 @@
+package handlers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/franela/play-with-docker/services"
+	"github.com/gorilla/mux"
+)
+
+func DeleteSession(rw http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	sessionId := vars["sessionId"]
+	log.Println(sessionId)
```

Comment:
I get the Code from GetSession, do you want me to also remove the print in GetSession ?